docs/modules_management.rst

```diff
@@ -0,0 +1,194 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Modules Management
+==================
+
+Airflow allows you to use your own Python modules in the DAG and in the Airflow configuration. The following article
+will describe how you can create your own module so that Airflow can load it correctly, as well as diagnose problems
+when modules are not loaded properly.
+
+This article is the last one for you if you need to adapt Airflow to the needs of your organization.
+
+Packages Loading in Python
+--------------------------
+
+The list of directories from which Python tries to load the module is given by the variable :any:`sys.path`. Python
+really tries to `intelligently determine the contents of <https://stackoverflow.com/a/38403654>`_ of this variable,
+including depending on the operating system and how Python is installed.
+
+You can check the contents of this variable for the current Python environment by running an interactive terminal as in
```

Comment:
The content of this variable varies depending on whether the interactive console is launched or the normal program is launched, and Airflow makes changes to it all. Finally, the content of this variable may differ from what the user will see and the Airflow will use. To address this problem, I added a new command `airflow info` that displays sys.path.
> Python PATH: [/usr/local/bin:/opt/airflow:/usr/local/lib/python37.zip:/usr/local/lib/python3.7:/usr/local/lib/python3.7/lib-dynload:/usr/local/lib/python3.7/site-packages:/files/dags:/root/airflow/config:/root/airflow/plugins]

If you want, you can also change the Python PATH label to sys.path to make the guide easier to read.

Reply:
I've added the details about airflow info and output of it in the airflow section of this doc. what do you think about that?